### ISSUE TITLE
Add possibility to change account when composing a toot

### ISSUE THREAD

**Issue by novadeviator** [tuskyapp/Tusky#558]
When sharing something from other applications the toot compose window assumes the last selected account and cannot be changed. It would be beneficial if account could be changed in the compose window. 

* * * *
- Tusky Version: 1.5.2
- Android Version:
- Android Device:
- Mastodon instance (if applicable):

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by charlag** [comment]
I don't think it is specific to sharing, we would like to add this functionality to the compose screen anyway, but later. It is indeed a required piece of functionality.
